snuba/web/rpc/__init__.py

```diff
@@ -66,6 +67,9 @@ def resolve(self, in_msg: Tin) -> Tout:
         raise NotImplementedError
 
 
+_TIME_PERIOD_BUCKETS = [1, 7, 14, 30, 90]
```

Comment:
```suggestion
_TIME_PERIOD_DAYS_BUCKETS = [1, 7, 14, 30, 90]
```

I assume the unit here is days?